tests/Resources/Page/HybridCloudConsole/Rhosak.robot

```diff
@@ -0,0 +1,123 @@
+*** Settings ***
+Library         SeleniumLibrary
+Resource        HCCLogin.robot
+
+
+*** Keywords ***
+Create Kafka Stream Instance
+  [Arguments]  ${stream_name}  ${stream_region}  ${cloud_provider}
+  Click Button  Create Kafka instance
+  Sleep  5
+  Maybe Accept Cookie Policy
+  Sleep  5
+  Maybe Agree RH Terms and Conditions
+  Wait Until Page Contains Element    xpath=//div[@id='modalCreateKafka']  timeout=10
+  ${warn_msg}=  Run Keyword And Return Status    Page Should Not Contain    To deploy a new instance, delete your existing one first
+  IF    ${warn_msg} == ${False}
+     Log  level=WARN  message=The next keywords are going to fail because you cannot create more than one stream at a time.
+  END
```

Comment:
You could skip the rest of the keyword?
https://robotframework.org/robotframework/latest/libraries/BuiltIn.html#Skip

Reply:
I will check this. My initial idea was to "skip" the rest of the TC, but then I thought that making the TC failing naturally (and giving just a warning) would implicitly test the limit on one kafka instance per user: if you get the warning message but the rest of the TC does not fail then there can be something wrong in the functionality.

What do you think?

Reply:
Is the limit of one kafka instance per user something that we really want to test? That seems like a feature of another product, so I wouldn't want to spend time debugging issues that are not in rhods. 
I also assume this limit of one instance per user is temporary / can be increased. This might need to be revisited down the line anyways.

Reply:
makes sense yeah. But my idea is that even if this is a functionality from another product it is still faced by RHODS users. So, given that this would be implicitly tested by our TC just without skipping a line which would fail anyway, it is costless for us to test it.

Reply:
I think we can throw error instead of failure or skip it will not affect our overall test suite status also when people will go thought the logs they can see the error.We should just skip the whole block of code below to stop the failure?

Reply:
@tarukumar when you say "throw error", you mean setting the log level to ERROR or doing something like "raise ValueError" in python?

Reply:
i was suggesting to set the log level to error.

Reply:
I set the log level to ERROR